app/components/AlarmBottomEdge.qml

```diff
@@ -65,6 +65,7 @@ BottomEdge {
     Component {
         id: editorPageBottomEdge
         AlarmPage {
+            onThemeChanged: { bottomEdge.commit(); bottomEdge.collapse(); }
```

Comment:
This is an hack as the BottomEdge is broken if the theme is changed after it  was displayed once, it can be fixed by "reviealing" and hiding it programmatically.